manage.py

```diff
@@ -296,9 +300,29 @@ def cf_startup():
         subprocess.Popen(['python', 'manage.py', 'update_schemas'])
 
 @manager.command
-def test_util():
-    df = efile_parser.get_dataframe(6)
-    efile_parser.parse_f3psummary_column_b(df)
+def load_efile_sheets():
```

Comment:
Let's add some in-line documentation and maybe a reference to this in the wiki so that people in the future will be able to find this.